pkg/cgroups/cgroups_test.go

```diff
@@ -0,0 +1,37 @@
+package cgroups
+
+import (
+	"testing"
+
+	"github.com/containers/podman/v2/pkg/rootless"
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func TestCreated(t *testing.T) {
+	// tests only works in rootless mode
+	if rootless.IsRootless() {
+		return
+	}
+
+	var resources spec.LinuxResources
+	cgr, err := New("machine.slice", &resources)
```

Comment:
should we check cgr is what we want?

Reply:
Sure, I was just throwing these in to keep the test checker happy. Since I don't see an easy way to test my changes.